plugins/timelib/timelib.cpp

```diff
@@ -568,33 +568,30 @@ TIMELIB_API void TIMELIB_CALL tlDateToString(size32_t &__lenResult, char* &__res
     __result = NULL;  // Return blank string on error
     __lenResult = 0;
 
-    // date is expected to be in form year*10000 + month * 100 + day, and must be later than 1900
-    // or we can't store it in a struct tm
+    // date is expected to be in form year*10000 + month * 100 + day
 
-    if (date >= 1900 * 10000)
```

Comment:
Is it worth retaining this test if date != 0 to avoid the condition in ConvertDateFormat? or does it potentially correspond to a sensible date?